aim/sdk/repo.py

```diff
@@ -515,6 +555,24 @@ def _prepare_runs_cache(self):
         db.init_cache(cache_name, db.runs, lambda run: run.hash)
         self.run_props_cache_hint = cache_name
 
+    def _delete_run(self, run_hash):
+        with self.structured_db:  # rollback db entity delete if subsequent actions fail.
+            # remove database entry
+            self.structured_db.delete_run(run_hash)
+
+            # remove data from index container
+            index_tree = self._get_index_container('meta', timeout=0).tree()
+            del index_tree.subtree(('meta', 'chunks'))[run_hash]
```

Comment:
`del tree[key]` is designed to remove the BLOBs as well.

Reply:
valid point, thanks!